metabulo/views.py

```diff
@@ -529,3 +530,39 @@ def get_pca_overview(validated_table):
     png_content = opencpu.generate_image('/metabulo/R/pca_overview_plot',
                                          validated_table.measurements)
     return Response(png_content, mimetype='image/png')
+
+
+@csv_bp.route('/csv/<uuid:csv_id>/analyses/wilcoxon', methods=['GET'])
+@use_kwargs({
+    'zero_method': fields.Str(missing='wilcox',
+                              validate=validate.OneOf(['wilcox', 'pratt', 'zsplit'])),
+    'alternative': fields.Str(missing='two-sided',
+                              validate=validate.OneOf(['two-sided', 'greater', 'less']))
+})
+@load_validated_csv_file
+def get_wilcoxon_test(validated_table: ValidatedMetaboliteTable,
+                      zero_method: str, alternative: str):
```

Comment:
Are `zero_method` and `alternative` meant to be constrained to the same list of values as in the validation above? In that case the `Literal` type would be good to put in here.

Reply:
Offloaded to https://github.com/girder/viime/issues/228